app/models/offspring.rb

```diff
@@ -0,0 +1,8 @@
+class Offspring < ApplicationRecord
+  belongs_to :user
+
+  class GradedOffspring < Offspring
```

Comment:
You need a new model...... Don't do this in the same file or you will end up mixing them and confusing everybody